cmd/api/src/model/filter.go

```diff
@@ -50,6 +50,7 @@ const (
 	FalseString    = "false"
 	IdString       = "id"
 	ObjectIdString = "objectid"
+	CollectedString = "collected"
```

Comment:
We have this defined as part of common properties so I would avoid defining this here. In line 107 you can instead use `common.Collected.String()`